docs/streams/developer-guide/write-streams.html

```diff
@@ -154,7 +154,7 @@ <h2>Using Kafka Streams within your application code<a class="headerlink" href="
       </div>
       <p>If there are other instances of this stream processing application running elsewhere (e.g., on another machine), Kafka
         Streams transparently re-assigns tasks from the existing instances to the new instance that you just started.
-        For more information, see <a class="reference internal" href="../architecture.html#streams_architecture_tasks"><span class="std std-ref">Stream Partitions and Tasks</span></a> and <a class="reference internal" href="../architecture.html#streams-architecture-threads"><span class="std std-ref">Threading Model</span></a>.</p>
+        For more information, see <a class="reference internal" href="../architecture.html#streams_architecture_tasks"><span class="std std-ref">Stream Partitions and Tasks</span></a> and <a class="reference internal" href="../architecture.html#streams_architecture_threads"><span class="std std-ref">Threading Model</span></a>.</p>
```

Comment:
fix the broken link from wrong `href="../architecture.html#streams-architecture-threads"` to `href="../architecture.html#streams_architecture_threads"` (the one with underscore)